packages/css/src/components/heading/README.md

```diff
@@ -2,15 +2,14 @@
 
 # Heading
 
-A heading conveys information about the content below it.
-Use headings to allow the user to grasp the structure of the page quickly.
+Communicates the organization of the content on a page.
 
 ## Guidelines
 
-- A heading describes the content beneath it.
```

Comment:
Without this item, we don't really describe what a Heading actually does. I'd leave it in.